### DISCUSSION TITLE
missing alerts in latest dev builds

### DISCUSSION THREAD

**Discussion by kirsch33** [blakeblackshear/frigate#14106] [General Support]
I fully understand with dev builds things are expected to break, and that's fine, but just as a heads up, I usually pull down dev builds every other day or so and I've noticed the past two days I haven't had any alerts being generated. After reviewing merged PRs it seems like the timing is in alignment to the gamma adjustment? I'm not sure, but happy to help track down. 

I can still find the events that should have been alerts under the motion section but they aren't even displaying as detections.

**Comment by NickM-27** [comment]
I'm not seeing this at all, it'd be very odd if the slight change in video output caused alerts to stop registering especially if detections continue to work.

**Reply by kirsch33** [reply]
Understood it is weird. And I was guessing on it being the gamma PR just due to timing. But on sept 29th every day up to it I had minimum 30+ alerts. Since then, it has been 0 and 1 with zero change in behavior (as in leaving for work, arriving, etc all the same). 

The same events are barely registering as motion instead of detections and therefore alerts. 

Again im happy to help debug with some guidance on what you want to see.

**Reply by NickM-27** [reply]
A video of the debug view would be good to see along with the config

**Reply by kirsch33** [reply]
i hope to have some time tonight to debug this but im still experiencing it 100%. starting from Monday this week, and each day after I have 30 alerts, 0, 30, and 0 again yesterday with no change in behavior. 

Its very strange but I'm hoping by enabling some debug logging and maybe exporting some footage of the same events for each day and setting up a camera to loop it, then recording the debug view will be helpful.

**Reply by kirsch33** [reply]
following up, this was just a weird coincidence and everything is fine. no issues with frigate. sorry for the false alarm.
